files/install-requirements.yaml

```diff
@@ -8,13 +8,17 @@
       - make
       - git
       - dnf-utils
+      - gcc
       - python3-pip
   - name: Install test rpm dependencies
     dnf:
       name:
       - python3-flexmock
       - python3-pytest
       - python3-pytest-cov
+      # These are needed for pip3 install .
+      - python3-devel
+      - krb5-devel
```

Comment:
you probably need this just because using pip for installation in next section or is there so other reason

Reply:
No, I have deleted. It was cause by dependencies which comes from rebase-helper. Now all is working properly.